src/main/clojure/clara/rules/engine.cljc

```diff
@@ -329,6 +329,7 @@
 (defrecord AlphaNode [env children activation]
```

Comment:
There are a lot of missing listener calls (a good thing to fix in a future commit). So I don't necessarily see this one as being any different than that.